rss2email/config.py

```diff
@@ -61,6 +83,36 @@ def setup_html2text(self, section='DEFAULT'):
             section, 'links-after-each-paragraph')
         _html2text.BODY_WIDTH = self.getint(section, 'body-width')
 
+    def set_overrides(self, overrides):
+        self.overrides = overrides
```

Comment:
This should be set in the constructor if possible.

Reply:
The reason I've done it this way is that you can pass in a pre-existing `Config` object to the constructor of `Feeds` but we still want to apply the overrides in that case, so we have to have a way to do it after construction. But actually the `config` argument is only used in the tests, which don't use any overrides, so we actually don't need this...

Reply:
It sounds overly complicated tbh :)